### DISCUSSION TITLE
What's the function to extract the last value of an array?

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#1333] [Q&A] [answered]
Hi,
how to extract the last value of an array?

If I have in example `echo 'input=ciao;bau;miao' | mlrgo --ojson put '$array=splita($input,";")'`, and I want to extract the last value of `array` field, how to extract from it, and put it in another new field?

Is there some `-1` way to do it?

Thank you

**Comment by johnkerl** [comment, marked as answer]
> Is there some -1 way to do it?

@aborruso yes, exactly:

```
$ echo 'input=ciao;bau;miao' | mlr --ojson put '$array=splita($input,";")'
[
{
  "input": "ciao;bau;miao",
  "array": ["ciao", "bau", "miao"]
}
]

$ echo 'input=ciao;bau;miao' | mlr --ojson put '$array=splita($input,";")[-1]'
[
{
  "input": "ciao;bau;miao",
  "array": "miao"
}
]
```

Please also see
https://miller.readthedocs.io/en/latest/reference-main-arrays/#negative-index-aliasing

**Reply by aborruso** [reply]
Thank you very much
